### PR TITLE
WIP: Implement exercise rna-transcription

### PR DESCRIPTION
**Pull request by samkrishna**
I'm implementing the RNA Transcription exercise. Should have something in a few days.

### CODE REVIEW

**Comment by samkrishna** [comment]
Code is done and submitted. Please let me know what the next steps are in the review process.

**Comment by samkrishna** [comment]
OK so here's what happened in the Travis CI build for the error to appear:

```
$ bin/configlet .
Evaluating .
-> An exercise with slug 'rna-transcription' is referenced in config.json, but no implementation was found.
-> An implementation for 'rna-trancription' was found, but config.json does not reference this exercise.
The command "bin/configlet ." exited with 1.
```

Any idea what I should do here? I've got the slug to match the folder name ('rna-transcription'), but I'm not sure where to go from here.

**Comment by masters3d** [comment]
Thank you for contributing @samkrishna
Looks like the check for the slug name is failing. Do you mind double checking this?

**Comment by samkrishna** [comment]
Thanks, @masters3d, that did the trick. There was a typo, but now it's fixed.

File: exercises/rna-transcription/RNATranscriptionExample.m [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+{
+    NSDictionary *dnaToRNADict = @{ @"G" : @"C",
+                                    @"C" : @"G",
+                                    @"T" : @"A",
+                                    @"A" : @"U" };
+    
+    NSArray *dnaKeys = [dnaToRNADict allKeys];
+    NSString *dnaKeyString = [dnaKeys componentsJoinedByString:@""];
+    NSCharacterSet *invertedSet = [[NSCharacterSet characterSetWithCharactersInString:dnaKeyString] invertedSet];
+    NSRange invertedSetMemberRange = [dnaStrand rangeOfCharacterFromSet:invertedSet];
```
Comment (samkrishna) [inline exercises/rna-transcription/RNATranscriptionExample.m]: Which variable?

**Comment by masters3d** [comment]
Thanks!

**Pull request merged**